CHANGELOG.md

```diff
@@ -9,6 +14,23 @@ All notable changes to this project will be documented in this file.
 - Scrollable history/bookmarks pages.
 - URL change check improved
 
+#### [v0.2.0](https://github.com/joshuef/peruse/compare/v0.10.0-1...v0.2.0)
```

Comment:
I don't think this is supposed to be added here, isn't it?

Reply:
Yeh, this whole bit was part of an auto-generator I was trying. I thought I'd deleted this (gigantic) section, (least I'd intended to). @hunterlester, would you be so rad as to remove this ginormous section from the changelog and we can get that into master with these changes.

Reply:
@hunterlester can you remove this 0.2.0 section and the list of commits, please.